examples/integrated_gradients_transformers.ipynb

```diff
@@ -0,0 +1,1179 @@
+{
```

Comment:
<p>"We train a convolutional neural network..." is not accurate, copied from the other notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/430/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Ya I did. Forgot to take it out.